libnd4j/buildnativeoperations.sh

```diff
@@ -154,6 +177,8 @@ if [ "$(uname)" == "Darwin" ]; then
 elif [ "$(expr substr $(uname -s) 1 5)" == "MINGW" ] || [ "$(expr substr $(uname -s) 1 4)" == "MSYS" ]; then
     HOST="windows"
     KERNEL="windows-x86_64"
+    # need to set build path separator, it ends up being wrong on msys2
+    BUILD_PATH_SEPARATOR=";"
```

Comment:
We can remove this. I fixed that in commit https://github.com/bytedeco/javacpp/commit/881ce8e7629488dbb1a9ee4433c13a5b76c231f1